lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -2593,16 +2593,22 @@ class DecomposeAtenArgMinMaxOp : public OpRewritePattern<OpTy> {
     // first the input tensor is flattened to 1d tensor and then the reduction
     // happens on the 0th dimension.
     if (isa<Torch::NoneType>(dim.getType())) {
-      BaseTensorType flattenType =
-          cast<BaseTensorType>(inputType.getWithSizesAndDtype(
-              {kUnknownSize}, inputType.getOptionalDtype()));
-      Value zero =
-          rewriter.create<ConstantIntOp>(loc, rewriter.getI64IntegerAttr(0));
-      Value end = rewriter.create<ConstantIntOp>(
-          loc, rewriter.getI64IntegerAttr(inputRank - 1));
+      Value zero = rewriter.create<ConstantIntOp>(loc, 0);
```

Comment:
Should the 0 be `rewriter.getI64IntegerAttr(0)`?

Reply:
The builder allows passing an integer instead of an integer attr.